pages/docs/reference/mpp-dsl-reference.md

```diff
@@ -270,6 +268,99 @@ kotlin {
 
 </div>
 
+For binaries configuration, the following parameters are available:
+
+|**Name**|**Description**| 
+| --- | --- |
+|`compilation`|The compilation from which the binary is built. By default, `test` binaries are based on the `test` compilation while other binaries - on the `main` compilation.|
+|`linkerOpts`|Options passed to a system linker during binary building.|
+|`baseName`|Custom base name for the output file. The final file name will be formed by adding system-dependent prefix and postfix to this base name.|
+|`entryPoint`|The entry point function for executable binaries. By default, it's `main()` in the root package.|
+|`outputFile`|Access to the output file.|
+|`linkTask`|Access to the link task.|
+|`runTask`|Access to the run task.|
```

Comment:
It is available for executables only. Also this property returns `null` if the binary target is other than `linuxX64`, `macosX64` or `mingwX64`.